packages/deployment/upgrade-test/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN . ./upgrade-test-scripts/start_to_to.sh
 
 ARG DEST_IMAGE
 #this is agoric-upgrade-10 / vaults
-FROM ${DEST_IMAGE} as agoric-upgrade-10
+FROM ghcr.io/agoric/agoric-sdk:34 as agoric-upgrade-10
```

Comment:
Note I updated this to use the published image for vaults. Please confirm this is the correct one for vaults?